packages/core-commands/src/admin-navigation-commands.js

```diff
@@ -25,9 +46,24 @@ export function useAdminNavigationCommands() {
 	useCommand( {
 		name: 'core/manage-reusable-blocks',
 		label: __( 'Manage all of my patterns' ),
-		callback: () => {
-			document.location.href = 'edit.php?post_type=wp_block';
+		callback: ( { close } ) => {
+			// __unstableIsBlockBasedTheme is not defined in the site editor so we need to check the context.
+			// If not the site editor and not a block based theme then redirect to the old wp-admin patterns page.
```

Comment:
Could we also add the `__unstableIsBlockBasedTheme` setting for the site editor, following something similar to the filter used to add it elsewhere?

Reply:
Logging out `isBlockTheme`, it looks like`__unstableIsBlockBasedTheme` is set when in the site editor. Is there something I'm missing here?

Reply:
ah, for future reference, my use of the term 'site editor' was too loose - I was refreshing the patterns page which is in the site editor, but doesn't actually have the site editor loaded at that point - will look and see if there is a way to get that setting in the same way the patterns hooks are extracting data from the site editor store.

Reply:
Had another look at this, but getting the `__unstableIsBlockBasedTheme` on the patterns page would involve adding extra dependencies that don't give any extra benefit over and above checking for `site-editor.php` in the path, which is an approach [already used in the command package](https://github.com/WordPress/gutenberg/blob/trunk/packages/core-commands/src/site-editor-navigation-commands.js#L124), so I think we should just stick with this for now.